src/jira/client/jira-client.ts

```diff
@@ -143,15 +144,15 @@ export const getJiraClient = async (
 		},
 		devinfo: {
 			branch: {
-				delete: (repositoryId: string, branchRef: string) =>
+				delete: (transformedRepositoryId: TransformedRepositoryId, branchRef: string) =>
```

Comment:
drag-n-drop replacement (you can highlight the new values with Control + F + "transformedRepositoryId" - no typos)